### ISSUE TITLE
Non jailbroken version of GC4iOS

### ISSUE THREAD

**Issue by Fadi5555** [WilliamLCobb/GC4iOS#17]
please can you give me the link to download GC4iOS for non jailbroken device please?

**Comment by MaquinaDelMisterio** [comment]
Sure. Go to-Code-Click on Download(Green button)-Download As Zip. No need to create new issues, read the already created ones, lot of info in there for further questions you may have.

**Comment by Fadi5555** [comment]
I already downloaded the zip file of this emu but i have no idea how to install it on my device
Can you help me please?

**Comment by MaquinaDelMisterio** [comment]
Of course. First: Need a Mac with Xcode. Second: Open GC4iOS.xcodeproject plug in your device. Third: Change the name of the app on build identifier and press on fix issue. Last: Press play button.

**Comment by Fadi5555** [comment]
Oh thanks a lot for helping but I don't have even mac to run Xcode so please can you give me this app like ppsspp method to install it on non device?

**Comment by MaquinaDelMisterio** [comment]
I would like to help, but i have a free dev account. What you can do, is, download the deb file and convert it to ipa and sign it, but you need dev certificates which can be made with xcode or pay some service to do that,

**Comment by nfsvsgta** [comment]
Relax will will update it after some tim

**Comment by ElijahZAwesome** [comment]
MaquinaDelMisterio where can I download the deb file? I'll keep checking the folders, but I'm jailbroken so I could just install the deb file.

**Comment by MaquinaDelMisterio** [comment]
http://www.williamlcobb.com/repo/ install it from here using Cydia. @ElijahZAwesome

**Comment by ElijahZAwesome** [comment]
@MaquinaDelMisterio thanks bruh, hope this emulator gets off the ground, it's got good potential

**Comment by MaquinaDelMisterio** [comment]
Welcome. Pal. Yes it is, hopefully it will, sometime.

**Comment by xowen1218x** [comment]
Where the deb file

**Comment by LeafyForLife** [comment]
You could use cydia impactor to download ipa files. Works on Windows, Linux, Mac, etc.

**Comment by xowen1218x** [comment]
Where's the IPA

**Comment by brand175** [comment]
can someone use ipwnstore from their jailbroken device and sign gc4ios and then upload the .ipa it here

**Comment by MaquinaDelMisterio** [comment]
You can use Impactor tool to sign apps from what i read on the web.

**Comment by brand175** [comment]
when you add gc4ios to the ipwnstore library you can be able to create an .ipa file without using xcode and be able to drag the .ipa to the desktop from your device (im on windows btw) the reason why i cant do it because i un jailbreaked my phone. Ive done this before but i deleted the gc4ios v1.2.1 .ipa from my computer and phone

**Comment by MaquinaDelMisterio** [comment]
This tool is free and can be used on Windows. Actually, this was made to install the ipa to jailbreak iOS 9.3

**Comment by brand175** [comment]
could you help me get the .ipa and then i will sign it on the cydia impactor

**Comment by MaquinaDelMisterio** [comment]
Sure, let me see where i have it.

**Comment by MaquinaDelMisterio** [comment]
There you go, sign it yourself with Impactor. https://www.dropbox.com/s/onwkmgqb2ao9cuh/GC4iOS.ipa?dl=0

**Comment by brand175** [comment]
thanks now i have gc4ios on ios 9.3.5  iphone 6 :3

**Comment by MaquinaDelMisterio** [comment]
You welcome, but im sorry to say that it wouldnt work.
